docs/source/contents/5_technical_metadata_properties.rst

```diff
@@ -0,0 +1,245 @@
+####################
+Technical Properties
+####################
+
+*****
+About
+*****
+
+While our MODS to RDF mapping aims to address how descriptive properties should be mapped in our next system, this
+section aims to explain how to map existing technical metadata properties our metadata properties that we need going
+forward.  When a technical metadata property may exist, its current mapping is described.
+
+**********
+Namespaces
+**********
+
++------------------------------+--------------+-------------------------------------------------------------+
+| Vocabulary Name              | Prefix       | URI                                                         |
++==============================+==============+=============================================================+
+| BIBFRAME                     | bf           | http://id.loc.gov/ontologies/bibframe/                      |
++------------------------------+--------------+-------------------------------------------------------------+
+| EBUCORE                      | ebu          | http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#width |
++------------------------------+--------------+-------------------------------------------------------------+
+| DCMI Metadata Terms          | dcterms      | http://purl.org/dc/terms/                                   |
++------------------------------+--------------+-------------------------------------------------------------+
+
+*******
+Mapping
+*******
+
+Duration
+========
+
+Use Case
+--------
+
+For audio / visual files, it is essential that we are able to measure the length of a the file that is associated with a
```

Comment:
Extra word on this line - get rid of "a".

Reply:
👍🏻